src/SkyCoords.jl

```diff
@@ -9,10 +9,71 @@ export AbstractSkyCoords,
        FK5Coords,
        separation
 
-import Base: convert
-
 include("types.jl")
 
+"""
+    str2rad(::AbstractString, force_ha=false)
```

Comment:
Since now this is not exported it's probably better to mention this as
```suggestion
    SkyCoords.str2rad(::AbstractString, force_ha=false)
```
I guess you'll need to do the same in the doctests below